src/EditorFeatures/Core/IntelliSense/QuickInfo/QuickInfoSourceProvider.QuickInfoSource.cs

```diff
@@ -37,25 +38,31 @@ private sealed class QuickInfoSource : IAsyncQuickInfoSource
             private readonly IAsynchronousOperationListener _asyncListener;
             private readonly Lazy<IStreamingFindUsagesPresenter> _streamingPresenter;
             private readonly IGlobalOptionService _globalOptions;
+            private readonly IInlineRenameService _inlineRenameService;
 
             public QuickInfoSource(
                 ITextBuffer subjectBuffer,
                 IThreadingContext threadingContext,
                 IUIThreadOperationExecutor operationExecutor,
                 IAsynchronousOperationListener asyncListener,
                 Lazy<IStreamingFindUsagesPresenter> streamingPresenter,
-                IGlobalOptionService globalOptions)
+                IGlobalOptionService globalOptions,
+                IInlineRenameService inlineRenameService)
             {
                 _subjectBuffer = subjectBuffer;
                 _threadingContext = threadingContext;
                 _operationExecutor = operationExecutor;
                 _asyncListener = asyncListener;
                 _streamingPresenter = streamingPresenter;
                 _globalOptions = globalOptions;
+                _inlineRenameService = inlineRenameService;
             }
 
             public async Task<IntellisenseQuickInfoItem> GetQuickInfoItemAsync(IAsyncQuickInfoSession session, CancellationToken cancellationToken)
             {
+                if (_globalOptions.GetOption(InlineRenameUIOptions.UseInlineAdornment) && _inlineRenameService.ActiveSession is not null)
+                    return null;
```

Comment:
consider doc'ing both sides to mention the other.

Reply:
also doc why we're doing this.  basically that quick info otherwise can go on top of hte rename dialog, blocking it from the user.